cache-config/t3c-apply/torequest/cmd.go

```diff
@@ -333,6 +336,22 @@ func checkRefs(cfg config.Cfg, cfgFile []byte, filesAdding []string) error {
 	return nil
 }
 
+//checkCert checks the validity of the ssl certificate
+func checkCert(c []byte) error {
+	block, _ := pem.Decode(c)
+	cert, err := x509.ParseCertificate(block.Bytes)
+	if err != nil {
+		return err
+	}
+	if cert.NotAfter.Unix() < time.Now().Unix() {
+		err = errors.New("Certificate expired: " + cert.NotAfter.Format("Jan 2, 2006 15:04 MST"))
```

Comment:
Nitpick: it'd be good to put this date string in a constant, especially since it's used twice